### ISSUE TITLE
Collection context: Review accessibility of live dom updates show/hide

### ISSUE THREAD

**Issue by mejackreed** [projectblacklight/arclight#910]
With new updates to implementation pattern we need to review the accessibility of the dom being updated and shown/hidden.

**Comment by seanaery** [comment]
Resolving this issue as part of a review of `arclight 1.0.x`. There no longer appear to be accessibility issues related specifically to DOM updates in v1.
